style.css

```diff
@@ -449,6 +449,84 @@ footer {
   display: none;
 }
 
+#detail {
+  background-color: #c1c7d0;
+  display: none;
+  position: fixed;
+  z-index: 9;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+}
+
+.detail {
+  border-radius: 16px;
+  background-color: #fff;
+  padding: 16px;
+  margin: 20px 16px 26px 16px;
+}
+
+.detail #mobile-img,
+#desktop-img {
+  width: 100%;
+  height: auto;
+}
+
+.detail #desktop-img {
+  display: none;
+}
```

Comment:
1. Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). However there are some few discrepancies as shown below as shown below :point_down: .  Kindly check and correct the positions of the `close button`, `see source` and `see live` buttons. The `boostrap` of the skills part is also out of position. 


### Your design 

![Screenshot from 2022-12-21 06-45-57](https://user-images.githubusercontent.com/65233860/208839917-46600c49-43f9-4992-b66f-1e93da507c17.png)



### Figma design

![Screenshot from 2022-12-21 06-45-44](https://user-images.githubusercontent.com/65233860/208839936-afd6765a-e450-4d72-9473-001c6c089f9a.png)